src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/integration/AddressResourceIntTest.kt

```diff
@@ -198,11 +204,64 @@ class AddressResourceIntTest : SqsIntegrationTestBase() {
       AddStandardAddressCheckRequest(caInfo, ppInfo, priority, uprn)
   }
 
```

Comment:
Realised I had forgotten to add tests for this endpoint when I added it.

Reply:
A shame unit tests were ok, but hibernate doesn't like it

  ........
    Caused by: org.hibernate.MappingException: Could not instantiate persister org.hibernate.persister.entity.JoinedSubclassEntityPersister

    Caused by: java.lang.IllegalArgumentException: Could not create type

    Caused by: java.lang.IncompatibleClassChangeError: class uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.entity.CurfewAddressCheckRequest$HibernateProxy$vhELgVIP cannot inherit from sealed class uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.entity.CurfewAddressCheckRequest